shared_steps/firstLogin.js

```diff
@@ -94,16 +95,18 @@ module.exports = {
 		return name;
 	},
 	logout: async function() {
-		let icon = await driver.$('[data-testid="initials"]');
-		await icon.click();
-		let logOut = await driver.$('[data-testid="logout"]');
-		await logOut.waitForDisplayed(3000);
-		await logOut.click();
-		let frontpageLoginBtn = await driver.$(Login.elem.frontpageLoginBtn);
-		await frontpageLoginBtn.waitForDisplayed(3000);
```

Comment:
At lot of commented out code. Row 98-104, 108.

Reply:
done